conf/imx_hab4/README.txt

```diff
@@ -0,0 +1,268 @@
+===========
+Documents:
+===========
+
+Feature Guide
+      read <https://www.nxp.com/docs/en/application-note/AN4581.pdf>
+
+Fuse Table:
+     <https://docs.google.com/spreadsheets/d/1gt2lW3NfYXsKdql7jOQn7ZdDgkhQAr1V/edit#gid=479826134>
```

Comment:
This link is private, probably better to just say that the fuse table is available in the reference security manual for the soc used.